Agenta-AI/agenta
# [UI]: implemented app management view

### Description
This PR significantly improves the App Management UI, enhancing user experience and workflow efficiency.

### Related Issue
Closes [AGE-914](https://linear.app/agenta/issue/AGE-914/implement-new-app-management-view).

### Key Changes
- Redesigned the app selection interface for a more intuitive user experience.
- A search bar was added to the app management page for easier apps navigation.

### QA Instructions
- **Create App:** Ensure the app creation process works smoothly through the UI.
- **Search Apps:** Test the search functionality to confirm you can easily find apps by name or other criteria.
